packages/fzf/build.sh

```diff
@@ -48,6 +49,15 @@ termux_step_make_install() {
 	mkdir -p $TERMUX_PREFIX/share/bash-completion/completions/
 	cp $TERMUX_PKG_SRCDIR/shell/completion.bash $TERMUX_PREFIX/share/bash-completion/completions/fzf
 
+	# Install zsh script:
+	mkdir -p $TERMUX_PREFIX/share/zsh/site-functions
+	cp $TERMUX_PKG_SRCDIR/shell/completion.zsh "$TERMUX_PREFIX"/share/zsh/site-functions/_fzf
+
+	# Install fish script:
+	mkdir -p $TERMUX_PREFIX/share/fish/vendor_functions.d
+	cp $TERMUX_PKG_SRCDIR/shell/key-bindings.fish "$TERMUX_PREFIX"/share/fish/vendor_functions.d/fzf_key_bindings.fish
+
+
 	# Install the rest of the shell scripts:
 	mkdir -p $TERMUX_PREFIX/share/fzf
 	cp $TERMUX_PKG_SRCDIR/shell/* $TERMUX_PREFIX/share/fzf/
```

Comment:
Shell completion files are already copied into prefix. Why not to just symlink (`ln -sfr src dest`) them?

Reply:
I tried to replicated aur script:
https://github.com/archlinux/svntogit-community/blob/packages/fzf/trunk/PKGBUILD